profile/README.md

```diff
@@ -0,0 +1,60 @@
+# Dart
+
+## A client-optimized language for fast apps on any platform
+
+Dart is:
+
+  * **Optimized for UI**: Develop with a programming language specialized     
+    around the needs of user interface creation.
+
+  * **Productive**: Make changes iteratively: use hot reload to see the result
+    instantly in your running app.
+  
+  * **Fast on all platforms**: Compile to ARM & x64 machine code for mobile,
+    desktop, and backend. Or compile to JavaScript for the web.
+
+Dart's flexible compiler technology lets you run Dart code in different ways,
+depending on your target platform and goals:
+
+  * **Dart Native**: For programs targeting devices (mobile, desktop, server,
+    and more), Dart Native includes both a Dart VM with JIT (just-in-time)
+    compilation and an AOT (ahead-of-time) compiler for producing machine
+    code.
+  
+  * **Dart Web**: For programs targeting the web, Dart Web includes both a
+    development time compiler (dartdevc) and a production time compiler
+    (dart2js).  
+
+![Dart platforms illustration](../assets/Dart-platforms.svg)
+
+## License & patents
+
+Dart is free and open source.
+
+See [LICENSE][license] and [PATENT_GRANT][patent_grant].
+
+## Using Dart
+
+Visit [dart.dev][website] to learn more about the [language][lang],
+[tools][tools], and to find [codelabs][codelabs].
+
+Browse [pub.dev][pubsite] for more packages and libraries contributed by the
+community and the Dart team.
+
+Our API reference documentation is published at [api.dart.dev][api], based on
+the stable release. (We also publish docs from our [beta][api-beta] and
+[dev][api-dev] channels, as well as from the [primary development
+branch][api-be]).
+
```

Comment:
Perhaps we also want to link to our security policy here?
```markdown
## Report vulnerabilities
To report vulnerabilities, please see our current security policy on https://dart.dev/security.
```

Reply:
Added